docs/quick-start.md

```diff
@@ -0,0 +1,29 @@
+## Quick Start
+
```

Comment:
content in quick start doesn't seem right. It should be the `curl` example. This is still very much ML specific. 

Are we planning to change this?

Reply:
Yes, this is what I have right now. Will update with curl example in the next version.